packages/main/src/themes/ColorPalette.css

```diff
@@ -2,10 +2,26 @@
 	display: inline-block;
 }
 
+.ui5-cp-root {
+	display: flex;
+	flex-direction: column;
+}
+
 .ui5-cp-item-container {
 	display: flex;
 	max-width: var(--_ui5_color-palette-row-width);
 	flex-flow: wrap;
 	max-height: var(--_ui5_color-palette-row-height);
 	overflow: hidden;
-}
\ No newline at end of file
+}
+
+.ui5-cp-separator {
+	height: 0.0625rem;
+    background: var(--sapContent_ForegroundBorderColor);
```

Comment:
Not sure about this one, but the separator color looks different from the one ine openui5

Here
<img width="640" alt="Screenshot 2021-02-23 at 12 03 08 PM" src="https://user-images.githubusercontent.com/15702139/108828305-5b936b00-75cf-11eb-996c-1c5873846463.png">


Openui5
<img width="637" alt="Screenshot 2021-02-23 at 12 03 20 PM" src="https://user-images.githubusercontent.com/15702139/108828330-63530f80-75cf-11eb-9d44-ac2f87f5a2c5.png">